client/state/analytics/actions/index.js

```diff
@@ -12,6 +12,7 @@ export {
 } from './record';
 export { recordTracksEventWithClientId, recordPageViewWithClientId } from './record-with-client-id';
 export { withAnalytics } from './with-analytics';
+export { enhanceWithSiteMainProduct } from './enhance-with-site-main-product';
```

Comment:
I wonder if this is a good opportunity to document these enhancers somewhere. They're not immediately obvious but are very useful companions of our analytics recording Redux actions.